delfin/exporter/prometheus/prometheus.py

```diff
@@ -41,26 +50,47 @@
      values={1600998817585: 20.264160223426305})]
 """
 
-unit_of_metric = {'response_time': 'ms', 'throughput': 'IOPS',
-                  'read_throughput': 'IOPS', 'write_throughput': 'IOPS',
-                  'bandwidth': 'MBps', 'read_bandwidth': 'MBps',
-                  'write_bandwidth': 'MBps'
-                  }
-
 
 class PrometheusExporter(object):
 
+    def __init__(self):
+        self.timestamp_offset_ms = self.set_timestamp_offset_from_utc_ms()
```

Comment:
Using timestamp offset may result in incorrect results in some cases.  (Eg. Day Light Saving changes in some regions)

Reply:
Prometheus uses Unix time internally when timestamp is not provided by exporter, same thing we are doing ..  for display Prometheus support timezone in display layer ..  https://prometheus.io/docs/introduction/faq/

Reply:
After initializing PrometheusExporter class, if day light saving changes, offset become wrong.

Reply:
i remember python time libraray has functions to determine if DST

Reply:
If we do not cache offset, and get it just before export, it will be wrong just one time when day light savings change

Reply:
done.. we get timezone info before every export now